docs/book/src/cronjob-tutorial/gvks.md

```diff
@@ -46,6 +46,22 @@ a package.
 Now that we have our terminology straight, we can *actually* create our
 API!
 
+## So, how can we create our API? 
+
+In the next section, [Adding a new API](./cronjob-tutorial/new-api.html) we will check how the tool help us to create our own API's with the command `kubebuilder create api`.
+
+The goal of this command is to create Custom Resource (CR) and Custom Resource Definition (CRD) for our Kind(s). To check it further see; [Extend the Kubernetes API with CustomResourceDefinitions][kubernetes-extend-api].
+
+## But, why create APIs at all?
+
+Our APIs and resouces represent our solutions on the clusters. Basically, the CRDs are a definition of our customized Objects, and the CRs are an instance of it.
```

Comment:
I think this section could use a bit more words.

```
New APIs are how we teach Kubernetes about our custom objects. The Go structs are used to generate a Custom Resource Definition (CRD) which includes the schema for our data as well as tracking data like what our new type is called. We can then create instances of our custom objects to use with (controllers)[some link todo].
```

Reply:
done